extensions/default/src/Toolbar/ToolbarLayoutSelector.tsx

```diff
@@ -59,11 +140,52 @@ function LayoutSelector({ rows, columns, className, onSelection, ...rest }) {
       rounded={rest.rounded}
       dropdownContent={
         DropdownContent !== null && (
-          <DropdownContent
-            rows={rows}
-            columns={columns}
-            onSelection={onSelection}
-          />
+          <div className="flex">
+            <div className="bg-secondary-dark flex flex-col gap-2.5 p-2">
+              <div className="text-aqua-pale text-xs">Common</div>
+
+              <div className="flex gap-4">
+                {commonPresets.map((preset, index) => (
+                  <LayoutPreset
+                    key={index}
+                    classNames="hover:bg-primary-dark group p-1"
+                    icon={preset.icon}
+                    commandOptions={preset.commandOptions}
+                    onSelection={onSelection}
+                  />
+                ))}
+              </div>
+
+              <div className="h-[2px] bg-black"></div>
+
+              <div className="text-aqua-pale text-xs">Advanced</div>
+
+              <div className="flex flex-col gap-2.5">
+                {advancedPresets.map((preset, index) => (
+                  <LayoutPreset
+                    key={index + commonPresets.length}
+                    classNames="hover:bg-primary-dark group flex gap-2 p-1"
+                    icon={preset.icon}
+                    title={preset.title}
```

Comment:
Is it possible to do something like the code below to avoid duplicating protocol names?
```
@@ -35,22 +35,22 @@ const defaultCommonPresets = [
 ];
 
 const defaultAdvancedPresets = [
-  { icon: 'layout-advanced-mpr', title: 'MPR', commandOptions: { protocolId: 'mpr' } },
+  { icon: 'layout-advanced-mpr', id: 'mpr', commandOptions: { protocolId: 'mpr' } },
   {
     icon: 'layout-advanced-axial-primary',
-    title: 'Axial Primary',
+    id: 'primaryAxial',
     commandOptions: { protocolId: 'primaryAxial' },
   },
   {
     icon: 'layout-advanced-3d-four-up',
-    title: '3D four up',
+    id: 'fourUp',
     commandOptions: { protocolId: 'fourUp' },
   },
-  { icon: 'layout-advanced-3d-main', title: '3D main', commandOptions: { protocolId: 'main3D' } },
-  { icon: 'layout-advanced-3d-only', title: '3D only', commandOptions: { protocolId: 'only3D' } },
+  { icon: 'layout-advanced-3d-main', id: 'main3D', commandOptions: { protocolId: 'main3D' } },
+  { icon: 'layout-advanced-3d-only', id: 'only3D', commandOptions: { protocolId: 'only3D' } },
   {
     icon: 'layout-advanced-3d-primary',
-    title: '3D primary',
+    id: 'primary3D',
     commandOptions: { protocolId: 'primary3D' },
   },
 ];
@@ -166,7 +166,7 @@ function LayoutSelector({
                     key={index + commonPresets.length}
                     classNames="hover:bg-primary-dark group flex gap-2 p-1"
                     icon={preset.icon}
-                    title={preset.title}
+                    title={servicesManager.services.hangingProtocolService.protocols.get(preset.id)['name']}
                     commandOptions={preset.commandOptions}
                     onSelection={onSelectionPreset}
                   />
```

Reply:
Are suggesting an id? The protocolId is kind of the same

Reply:
The protocols already have a "name" field:
```
export const fourUp = {
  id: 'fourUp',
  locked: true,
  name: 'fourUp',  // <- this one
  ...
}   
```
I suggested to use the 'name' field from the protocol defenition instead of declaring new 'title' field in [ToolbarLayoutSelector.tsx](https://github.com/OHIF/Viewers/pull/3923/files#diff-934bb0ff11f5013f88de1c70711e535c6323283fa55b157d3a9a05421cd12153)

And in the code below, you can remove the "title" field:
```
  {
    icon: 'layout-advanced-3d-four-up',
    title: '3D four up',
    commandOptions: { protocolId: 'fourUp' },
  },
```
But you are most likely more experienced and it is better for you to make the decision

Reply:
ooooh, yes it makes sense, @IbrahimCSAE